src/binding.cc

```diff
@@ -2029,6 +2034,38 @@ MaybeBool v8__Module__SetSyntheticModuleExport(const v8::Module& self,
       isolate, ptr_to_local(export_name), ptr_to_local(export_value)));
 }
 
+struct WasmStreamingSharedPtr {
```

Comment:
We do not usually wrap a shared_ptr in a struct. Why is that necessary here?

Reply:
It's not strictly necessary but I like the newtype idiom. It makes it harder to use it wrong.

It's also nice to have the same type name in C++ and Rust, easier to grep.